### ISSUE TITLE
Proposed new structure for the templates

### ISSUE THREAD

**Issue by jenniferward** [rism-ch/muscat#566]
For the template selection: http://muscat.rism.info/admin/sources/select_new_template?locale=en
I would like to suggest a clearer organization as follows, with headings, a hierarchy, and new terminology. Note that the links are identical for the 2nd and 3rd templates under Music manuscript, and for the 1st and 3rd templates under Printed music. I think listing the links separately within a hierarchy is clearer from a user's perspective than trying to explain the logic of combining two different functions in one (currently beta has "Musikdruck, Haupteintrag Sammlung oder Individualeintrag" and "Musikhandschrift, Indvidual- oder Teileintrag").

**Music manuscript**
Manuscript collection parent record [link to type=000_collection]
⋅⋅⋅Item within a collective manuscript [link to type=002_source]
Single music manuscript [link to type=002_source]

**Printed music**
Printed collection parent record [link to type=011_edition]
⋅⋅⋅Item within a printed collection [link to type=013_edition_content]
Single printed music edition [link to type=011_edition]

**Libretto**
Handwritten libretto
Printed libretto

**Treatise**
Handwritten treatise
Printed treatise

⋅⋅⋅ = please indent here.

**Comment by xhero** [comment]
Should we do this in 3.7 or postpone it?

**Comment by HirschSt** [comment]
Indeed that would be an improvement, but I think we just cannot make this for 3.7, so we should postpone this for 3.7.1

**Comment by lpugin** [comment]
This is what we had in Muscat 2 and it was requested to have it simplified. I am not against it but it should not be changed without further discussion.

**Comment by jenniferward** [comment]
At the very least, the English terms for printed music absolutely must be changed before the templates are released. Based on descriptive terms in the German version, I propose:

Edition* --> Printed music edition, collection parent record or individual item
Music imprint --> Printed music edition, item in collection


Can we also change the "Collection" label so it's clear it has to do with music manuscripts?

Collection--> Music manuscript, collection parent record

**Comment by HirschSt** [comment]
Nothing against relabeling, @lpugin ok?

**Comment by lpugin** [comment]
I am not. But I remember that these terms were chosen after a discussion about the current status of the data, and namely the we currently have prints with the template manuscript template, and for this reason the neutral term "Collection" was chosen. Just for the record.

**Comment by HirschSt** [comment]
@lpugin definitvely correct; since template selection is done with creating new items AFAIK I think changing the label now is not highly problematic **IMO**

**Comment by lpugin** [comment]
> since template selection is done with creating new items AFAIK I think changing the label now is not highly problematic IMO

This makes sense.

**Comment by xhero** [comment]
What should we do with this issue?

**Comment by HirschSt** [comment]
IMO relabeling is done, so this issue should be closed.

**Comment by jenniferward** [comment]
We are meeting next week to discuss this and will update this space by March 1. 
A full structure would have to include an option for printed treatises and libretti that occur within printed collections but templates for those kinds of sources are currently missing (see #722).

**Comment by MFalletta** [comment]
Hier ein Vorschlag für die neue Anordnung der Templates. Ist eine grafische Lösung machbar?
[Anordnung Templates.pdf](https://github.com/rism-ch/muscat/files/2919073/Anordnung.Templates.pdf)

Ausgangspunkt ist das Konvolut, da es mit allen Materialien kombiniert vorkommen kann.

**Comment by jenniferward** [comment]
Here is the English terminology, which is a bit more wordy. I think some definitions are necessary but I don't know if those should go in the template selection, where it is readily visible, or in the guidelines.

https://docs.google.com/document/d/1t6-4ed8t1opmEE2tK5S0b3HjoidUR0W8YR0j0zFFrwU/edit?usp=sharing  

I've included the template numbers that the templates should link to. I am assuming that the "Music manuscript collection parent record" is essentially turning into a record for any kind of handwritten collection, whether music, libretto, or treatise. Or should there be separate collection records for libretti and treatises? (Neither have keys, for example.)

It might also be good to link to examples of the different kinds of entries, but again maybe this is better off in the guidelines.

**Comment by jenniferward** [comment]
If this template https://muscat.rism.info/admin/sources/new?new_type=000_collection stays as a template for all 3 kinds of handwritten sources (MSS, treatises, libretti), then I think it needs to be renamed.

Not New Source - Music manuscript, collection parent record
But how about:
New Source - Collection parent record for handwritten music, treatises, or libretti

? Too wordy?

**Comment by xhero** [comment]
That is perfect for me.
I'm just wondering if we need two new record_types for treatises and libretti **not** in collections. For sources we do not have a separate template, a source not in a collection is simply not inserted in one (and has no 773). This is referred to the lines in your doc:

* Individual entry in a printed libretto collection [links to NEW]
* Individual entry in a printed treatise collection [links to NEW]

Would templates 015 and 017 suffice (these are record_type 5 and 7 in the internal format)?

**Comment by jenniferward** [comment]
Yes, I think that's right.

**Comment by xhero** [comment]
I _also_ propose to make the MARC template number match the internal record_type....

**Comment by jenniferward** [comment]
I found some small things to change on the "Select new template" screen - if they can be squeezed in for 5.2, that'd be great:

1. For consistency, let's make the headings plural:
Printed music--> Printed music editions
Treatise, handwritten --> Treatises, handwritten
Treatise, printed --> Treatises, printed

2. In the "Help" text, it would be better to have the paragraph about "A Composite volume is a volume that..." at the end instead of at the beginning.

**Comment by HirschSt** [comment]
done with https://github.com/rism-ch/muscat/commit/a77573f2c78458ae5f107d18bcaca58819ffddb5
